codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -235,10 +235,12 @@ private void writePendingBytes() {
     }
 
     /**
-     * This will allocate bytes by stream weight and priority for the entire tree rooted at {@code parent}, but does
-     * not write any bytes.
+     * This will allocate bytes by stream weight and priority for the entire tree rooted at {@code parent}, but does not
+     * write any bytes. Priority is generally distributed proportionally to siblings in the priority tree, however we
```

Comment:
`Priority is generally distributed proportionally`

This is a bit confusing because we are not distributing `priority`.

What about:

`The connection window is generally distributed amongst siblings according to priority, however ...`